tests/button/buttonview.js

```diff
@@ -66,30 +67,30 @@ describe( 'ButtonView', () => {
 			} );
 		} );
 
-		describe( 'title', () => {
+		describe( 'tooltip', () => {
 			it( 'is not initially set ', () => {
-				expect( view.element.attributes.title ).to.undefined;
+				expect( view.element.attributes[ 'data-ck-tooltip' ] ).to.undefined;
```

Comment:
You probably know that but it is possible to set and get `data-*` by `element.dataset`, `view.element.dataset.ckTooltip`